src/TranslationServer/Model/Repository.php

```diff
@@ -105,7 +105,7 @@ public static function createByFilePath($filepath)
         self::createPlainRepresentationByArray(
             $translations,
             $language,
-            $data,
+            ($data) ? $data : [],
```

Comment:
Yes I've start with a is_array($data) in case of data was a string, but i'ts not possible.
Will fix it